packages/controllers/organization_detail.coffee

```diff
@@ -0,0 +1,13 @@
+if Meteor.isClient
+  Template.organizationDetail.onCreated ->
+    @subscribe('organizationDetail', @data.organizationId)
+
+  Template.organizationDetail.helpers
+    organization: ->
+      Organizations.findOne(@organizationId)
+    format: (description)->
+      description?.split(/\r?\n\n/g)
```

Comment:
Yeah, we already have 2 and I'm sure there will be other instances. Where should I put it?